pantheon-bundle/src/main/java/com/redhat/pantheon/model/api/Child.java

```diff
@@ -37,28 +41,92 @@
      * Indicates if the child exists.
      * @return True, if the child exists. False otherwise.
      */
-    boolean isPresent();
+    default boolean isPresent() {
+        return get() != null;
+    }
 
     /**
-     * Provides a null-safe way to operate on the value of the child, and return an
-     * {@link Optional} with the result of the operation. This allowes the caller to
-     * continue to operate in a null-safe fashion.
-     * @param func The function to apply to the value
+     * Convert this Child to an {@link Optional}
+     * @return An {@link Optional} with the contained value.
+     */
+    default Optional<T> asOptional() {
+        return Optional.ofNullable(get());
+    }
+
+    /**
+     * Navigates to the {@link Child} provided by an accessor function.
```

Comment:
```suggestion
     * Navigates to the {@link Child} provided by an accessor function.
     * Uses the Null Object pattern to avoid throwing an NPE in a long chain of navigation.
     * For example:
     * Child.from(A)
     * .toChild(B)
     * .toChild(C)
     * .toChild(D)
     * .toChild(E)
     * .getAsOptional()
     * If C does not exist, the chain will not throw an exception because the nullChild will be returned for the remainder of the calls.
```